packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/bin/server/controllers/authentication_controller.dart

```diff
@@ -31,6 +31,11 @@ class AuthenticationController extends ApiController {
       '/api/logout',
       _logoutHandler,
     );
+    router.addRequest(
+      RequestType.POST,
+      '/api/auth/validate-token',
```

Comment:
In the OAuth2 specification there is no such think as `validate-token`

Reply:
removed